src/views/count.vue

```diff
@@ -115,6 +121,13 @@
         } else {
           showToast(translate("Enter the stock count for the product"))
         }
+      },
+      fetchProduct(sku: any) {
+        const payload = {
+          currentSku: sku,
+          queryString: '*' + sku + '*'
```

Comment:
Why we are passing SKU twice, we can pass a single value and add * in the actions part, WDYT?

Reply:
Yes, Sir, I changed the code to the same. And we are adding * in the action now.
Thank You sir for reviewing my code.